masonite/providers/HelpersProvider.py

```diff
@@ -16,7 +17,7 @@ class HelpersProvider(ServiceProvider):
     def register(self):
         pass
 
-    def boot(self, view: View, request: Request):
+    def boot(self, view: View, request: Request, manager: MailManager):
```

Comment:
I feel it should not be a `manager`, it is quite generic.

Reply:
probably right although in the current context it's pretty clear what it's doing. If we need to add a second manager we'll have to refactor a bit